.github/workflows/integrate.yaml

```diff
@@ -78,13 +86,14 @@ jobs:
     - name: Run test
       run: |
         sg microk8s -c "microk8s enable metallb:10.64.140.43-10.64.140.49"
+        cd charms/${{ matrix.charm }}
```

Comment:
This test is only at the bundle level, not the charm level.  But if we had charm-level integration tests, completely agree with this